I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 10, in script
    $variable = input_ut()
  File "game/script.rpy", line 10, in <module>
    $variable = input_ut()
  File "game/System.rpy", line 17, in input_ut
    your_text = input("Blaj")
EOFError: EOF when reading a line

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 10, in script
    $variable = input_ut()
  File "D:\Programms\Renpy\renpy-6.99.14.1-sdk\renpy\ast.py", line 848, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "D:\Programms\Renpy\renpy-6.99.14.1-sdk\renpy\python.py", line 1812, in py_exec_bytecode
    exec bytecode in globals, locals
  File "game/script.rpy", line 10, in <module>
    $variable = input_ut()
  File "game/System.rpy", line 17, in input_ut
    your_text = input("Blaj")
EOFError: EOF when reading a line

Windows-7-6.1.7601-SP1
Ren'Py 6.99.14.1.3218
 
Fri Feb 01 19:12:18 2019
